pkg/proxy/ipvs/proxier.go

```diff
@@ -106,6 +106,7 @@ var iptablesJumpChain = []struct {
 	{utiliptables.TableNAT, utiliptables.ChainPrerouting, kubeServicesChain, "kubernetes service portals"},
 	{utiliptables.TableNAT, utiliptables.ChainPostrouting, kubePostroutingChain, "kubernetes postrouting rules"},
 	{utiliptables.TableFilter, utiliptables.ChainForward, KubeForwardChain, "kubernetes forwarding rules"},
+	{utiliptables.TableFilter, utiliptables.ChainInput, KubeNodePortChain, "kubernetes health check rules"},
```

Comment:
I'm less familiar with the IPVS rules but I believe the comment I made about iptables should apply here too

Reply:
yeah same reason as my previous [comment](https://github.com/kubernetes/kubernetes/pull/97824#discussion_r568863008) why we need to keep this rule.